util/file.go

```diff
@@ -0,0 +1,44 @@
+package util
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/pathfinder-cm/pathfinder-go-client/pfmodel"
+)
+
+func WriteStringToFile(filename string, data string) (*os.File, error) {
+	file, err := os.Create(filename)
+	if err != nil {
+		return file, err
+	}
+	defer file.Close()
+
+	_, err = io.WriteString(file, data)
+	if err != nil {
+		return file, err
+	}
+	return file, nil
+}
+
+func SetupBootstrapFileContent(bs pfmodel.Bootstrapper, content string, mode int) (string, int) {
+	switch bs.Type {
+	case "chef-solo":
+		content = `
```

Comment:
can we put this content into an external template file instead and not hardcode it here?